addons/mail/models/ir_attachment.py

```diff
@@ -4,14 +4,14 @@
 
 from odoo import _, models, SUPERUSER_ID
 from odoo.exceptions import AccessError, MissingError, UserError
-from odoo.tools import consteq
+from odoo.tools.misc import verify_limited_field_access_token
 from odoo.addons.mail.tools.discuss import Store
 
 
 class IrAttachment(models.Model):
     _inherit = 'ir.attachment'
 
-    def _check_attachments_access(self, attachment_tokens):
+    def _check_attachments_ownership(self, attachment_tokens):
```

Comment:
Can we take the opportunity to change the API of this method?

I'd much prefer if it returned True/False instead, possibly renamed to `_has_attachments_ownership`.

And then caller is responsible to check the value and raise accordingly (not found in controllers).

The current way of writing this method is very error prone, as a missing case/missing raise will be very hard to detect. Whereas if the method returns False at the end, it's very easy to detect when True is actually returned.